packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +863,93 @@ describe('serverless cluster', () => {
 
 
   });
+
+  test('can create a Serverless cluster without vpc', () => {
+    // GIVEN
+    const stack = testStack();
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_MYSQL,
+    });
+
+    // THEN
+    expect(stack).toHaveResourceLike('AWS::RDS::DBCluster', {
+      Engine: 'aurora-mysql',
+      EngineMode: 'serverless',
+    });
+    // Expect DbSubnetGroupName and VpcSecurityGroupIds are not defined.
+    // It can't be done with toHaveResourceLike("", { DbSubnetGroupName: undefined })
```

Comment:
You can use the `ABSENT` helper: https://github.com/aws/aws-cdk/blob/1aa1588640b34be1dc027f79e3b9dd90e9b8b26f/packages/%40aws-cdk/aws-rds/test/instance.test.ts#L313-L318